modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -93,16 +101,17 @@ This procedure creates these tools channels:
 |===
 
 | OS Version | Base Channel                     | Tools Channel
-| {es} 7     | RHEL Expanded Support 7          | RES7-SUSE-Manager-Tools x86_64
-| {es} 8     | RHEL or SLES ES or CentOS 8 Base | RES8-Manager-Tools-Pool for x86_64 and RES8-Manager-Tools-Updates for x86_64
+| {sll} 7     | RHEL Expanded Support 7          | RES7-SUSE-Manager-Tools x86_64
+| {sll} 8     | RHEL or SLES ES or CentOS 8 Base | RES8-Manager-Tools-Pool for x86_64 and RES8-Manager-Tools-Updates for x86_64
+| {sll} 9     | RHEL and Liberty 9 Base          | RES9-Manager-Tools-Pool for x86_64 and RES9-Manager-Tools-Updates for x86_64
```

Comment:
Are we sure we use RES9 in the tools channel? I thought it was renamed to EL9

Reply:
the update channel is on the liberty linux extension product and named `SLL-9-Updates for x86_64`

Reply:
https://suma-bv-43-srv.mgr.prv.suse.net/rhn/manager/admin/setup/products has an unfinished installation since a week or longer.  I'll now use
```
| {sll} 9     | RHEL and Liberty 9 Base          | EL9-Manager-Tools-Pool for x86_64 and SLL9-Manager-Tools-Updates for x86_64
```
If this is not correct please provide a c&p variant.

Reply:
the channel names as they are right now on my fresh setup are `EL9-Manager-Tools-Pool for x86_64` and `EL9-Manager-Tools-Updates for x86_64`

Reply:
@lucidd @keichwa should we mention el8-uyuni-client-x86_64, el9-uyuni-client-x86_64 somewhere? Please give another look to this page. Still seems not good.
For more context, follow this thread https://suse.slack.com/archives/C02D12TNYLS/p1673872934090469?thread_ts=1673872675.415999&cid=C02D12TNYLS please

Reply:
The uyuni channels cannot be added with this page. You can only add them with spacewalk-common-channels tool which is described for uyuni. AFAIK we have this there.

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1205719#c4